### PR TITLE
Adds support for opening local documentation with shortcut control+shift+f1

### PR DESCRIPTION
**Pull request by ashleygrobler04**
Hi there. I've added the option to press control+shift+f1 to open offline documentation for the program. I've also modified the file for ci to include the documentation when building. Hopefully this runs correctly.

### CODE REVIEW

File: doc/readme.html
Comment (ogomez92) [inline doc/readme.html:1]: This file shouldn't be included.  It should be generated via pandoc or something.

File: src/main.py [hunk 1]
```diff
@@ -28,7 +28,9 @@
 try:
     handler = WXKeyboardHandler(uiManager.frame)
     handler.register_key("control+alt+space", uiManager.toggle_visibility)
-    handler.register_key("control+alt+r", windowManager.hide_currently_focused_window)
+    handler.register_key(
+        "control+alt+r", windowManager.hide_currently_focused_window)
+    handler.register_key("control+shift+f1", uiManager.openDocs)
```
Comment (ogomez92) [inline src/main.py:33]: I don't like the fact that this is a global hotkey. We can ad a help button in the UI, but I don't feel like adding yet another global hotkey even when the program is not in focus just to open documentation is a good idea.

File: src/managers/ui_manager.py [hunk 2]
```diff
@@ -249,10 +252,18 @@ def new_snippet_button_clicked(self, event):
...
         if event.GetKeyCode() == wx.WXK_ESCAPE or event.GetKeyCode() == wx.WXK_F4 and event.AltDown():
             self.frame.Hide()
             return
-            
-        event.Skip()
\ No newline at end of file
+
+        event.Skip()
+
+    def openDocs(self):
```
Comment (ogomez92) [inline src/managers/ui_manager.py:262]: Can we create another button to trigger this function?

**Review by ogomez92 — CHANGES_REQUESTED** [review]

**Comment by ashleygrobler04** [comment]
Hi there. I've generated the documentation with markdown, I hope that is fine. If I need to use pandoc, I will do so...
With regards to the help button in the UI, I can do that.

**Comment by ashleygrobler04** [comment]
When you say that the file should be generated and not included, do you meen this should happen at build time or what? Sorry if I don't understand correctly...

**Comment by ogomez92** [comment]
In theory yes. But I'm thinking that ci on Windows will not have
pandoc installed and  I'm not sure how hard it will be to install it.
Maybe we can leave the file here and include it.

**Comment by ashleygrobler04** [comment]
What I thought of doing was just to let webbrowser open up a URL to the readme.md file. This would make life a lot easier for ci then and it's a bit easier to implement. Will that be fine?
I know I said this adds offline support but these days everyone has internet.

**Comment by ashleygrobler04** [comment]
This way, when you update the readme file to maybe add something new, it will reflect those changes to the program.

**Review by ogomez-at-wiris — APPROVED** [review]
I will most likely change this to point to a readme on my website, not sure yet. merging for now thanks.

**Pull request merged**